routes/graphql-routes.js

```diff
@@ -2,20 +2,94 @@
 
 import Router from 'koa-router'
 import graphqlHTTP from 'koa-graphql'
+import DataLoader from 'dataloader'
 import jwt from 'koa-jwt'
+import schema from '../schema/index'
+import db from '../models/db'
+import bcrypt from 'bcrypt'
 
 const router = new Router()
-let jwtMiddleware = jwt({
+
+const jwtMiddleware = jwt({
     secret: process.env.JWT_SECRET
 })
 
-router.post('/graphql', jwtMiddleware, async (ctx) => {
+/* router.post('/graphql', jwtMiddleware, async (ctx) => {
```

Comment:
Pls remove if it's not needed anymore or place a relavant comment to understand why it is commented.